zebra/zebra_vxlan.c

```diff
@@ -1961,6 +1961,7 @@ static int zvni_local_neigh_update(zebra_vni_t *zvni,
 	char buf2[INET6_ADDRSTRLEN];
 	zebra_neigh_t *n = NULL;
 	zebra_mac_t *zmac = NULL, *old_zmac = NULL;
+	bool check_rbit = false;
```

Comment:
This is good. As the new boolean variable introduced now is actually used to control the sending update. SO if possible, please change the name to appropriate name.  

I am approving the request.